torch_ipex/csrc/cpu/DevOPs.cpp

```diff
@@ -50,16 +50,30 @@ at::Tensor AtenIpexCPUDev::dil_convolution(
   CHECK_DNNL_OP_PRE_COND(weight);
 
   dbl::comm::reorder_to_bf16_for_mix_prec(input);
-  dbl::comm::reorder_to_bf16_for_mix_prec(weight);
```

Comment:
I hope we can keep this reorder because it’s dealing with type conversion, and let the prepacking only dealing with format conversion.

Though it would emit two reorders at the first call to bf16 conv, one for f32->bf16, one for abcd->ABcdXXXXXX, this would make prepacking condition simple: only when the format is public, do we have to perform the format conversion.